hc/front/forms.py

```diff
@@ -41,3 +42,16 @@ class AddWebhookForm(forms.Form):
 
     def get_value(self):
         return "{value_down}\n{value_up}".format(**self.cleaned_data)
+
+
+class CreateCategoryForm(forms.ModelForm):
+    '''form to cater for the creation of a category for a blog'''
+    class Meta:
+        model = Category
+        fields = ['name']
+
+class AddBlogPostForm(forms.ModelForm):
+    '''Form to cater to the creation of Blog entires'''
+    class Meta:
+        model = Blog
+        fields = ['title', 'body', 'category']
```

Comment:
there should be a new line here too

Reply:
This is well noted. Let me effect the changes. I will ping you once completed.